app/controllers/webhooks_controller.rb

```diff
@@ -48,6 +48,23 @@ def mailjet
     success
   end
 
+  def mailpace
+    # see https://docs.mailpace.com/guide/webhooks#email-events
+
+    message_id = Email::MessageIdService.message_id_clean(params["message_id"])
+    to_address = params["to"]
+    status = params["Type"]
```

Comment:
Are you sure this is the right key? According to the documentation you linked in the comment above, I don't see any `Type` key, but a `status` one. Is that what you meant?

Reply:
You're right. Crap. I'm used to using PHPStorm which autosave. I used VSCodium this time. I've changed it locally but haven't saved this change. thx for the hint!